lua/cinema/misc/cl_weaponvolume.lua

```diff
@@ -0,0 +1,23 @@
+-- This file is subject to copyright - contact swampservers@gmail.com for more information.
+local swamp_weaponvolume = CreateClientConVar("swamp_weaponvolume", "1", true, false, "", 0.0, 1.0)
+
+local weapon_sound_patterns = {
+    "weapon",
+    "bullet",
+    "garand",
+    "throatneck",
+    "isissong"
+}
+
+hook.Add("EntityEmitSound", "EntityEmitSound_WeaponVolume", function(data)
+    for k, v in pairs(data) do
```

Comment:
Remove the loop and access the SoundName key directly like ``local sound = data.SoundName``

Reply:
I see you beat me to it